crates/flycheck/src/lib.rs

```diff
@@ -128,7 +128,7 @@ struct FlycheckActor {
 
 enum Event {
     Restart(Restart),
-    CheckEvent(Option<cargo_metadata::Message>),
+    CheckEvent(Option<Box<cargo_metadata::Message>>),
```

Comment:
This is the most common variant, so using a `Box` here wil actually increase the  average memory usage.

Reply:
removed this commit